examples/syn-flood/README.md

```diff
@@ -14,9 +14,15 @@ The example is located in the `examples/syn-flood` sub-directory. To run the app
 /examples/syn-flood$ cargo run -- -f syn-flood.toml
 ```
 
+To observe the `SYN` flood traffic, in the vagrant VM, run `tcpdump` to capture packets seen by `eth3`,
```

Comment:
minor: link to the vm readme information so people know where this came from.

Reply:
@drunkirishcoder

Reply:
will change the wording to make it more clear.